### DISCUSSION TITLE
Asset Inventory: Asset Data Requirements - Asset key skills and knowledge along with names and contact details – T

### DISCUSSION THREAD

**Discussion by sreyac** [CriticalAssetManagement/CAMS#120] [General]
Need further clarification on what this is

**Comment by luke-feeney** [comment]
Not just who is responsible - who is skilled operator for this asset 

Name and contact

**Reply by luke-feeney** [reply]
prompt to update every 6 months - as responsible moves on (post-MVP) 

organizations and agency
